roles/couchbase/tasks/couchbase_init.yml

```diff
@@ -0,0 +1,229 @@
+---
+- name: wait
+  pause: 
+    seconds: 30
```

Comment:
This needs to be more clearly defined. Why are we waiting? If we need to wait on a task before proceeding to the next, we need to use a conditional and retry logic. That will allow us to optimally wait on something that could perhaps be under 30 seconds.

Reply:
added more explicit description of wait